virttest/iscsi.py

```diff
@@ -717,22 +731,21 @@ def export_target(self):
             #     (lvcreate -name lv_iscsi -size 1G vg)
             # 2) Create a fileio backstore,
             #    which enables the local file system cache.
-            #
-            # This class Only works for emulated iscsi device,
-            # So fileio backstore is enough and safe.
 
             # Create a fileio backstore
```

Comment:
```suggestion
            # Create a backstore
```
Here, not only fileio backstore but also other kinds of backstore according to `self.iscsi_backend`

Reply:
updated